### DISCUSSION TITLE
Help to Build Armv7

### DISCUSSION THREAD

**Discussion by wgarnett** [JACoders/OpenJK#1100] [Q&A]
Hello ....

I Need OpenJo , Not OpenJK , Armv7 version .....
plz help to build and thanks again ......

**Comment by ensiform** [comment]
What have you tried so far? What is the issue with building? What platform for armv7? Arm platforms are not out of the box experience. Also armv7 may not even have full OpenGL support which is necessary.

**Reply by Calinou** [reply]
> Also armv7 may not even have full OpenGL support which is necessary.

[gl4es](https://github.com/ptitSeb/gl4es) may be usable here. Maybe even [box86](https://github.com/ptitSeb/box86) to run the x86 binary directly on the ARM device in question?

**Reply by ensiform** [reply]
For reference, #1101 has more information about the system in question since a new issue was reopened.

**Comment by wgarnett** [comment]
here is arm openjk =>  https://github.com/krishenriksen/OpenJK

I need arm openjo , plz help to use above to build one and share , thanks ..

**Reply by ensiform** [reply]
You'll need to figure out how to contact Kris then and ask him to make a version of his code for the outcast sp (OpenJO) under releases.

There's no outcast MP support in openjk/jo.
